scripts/bin/fks-moveToFykos.sh

```diff
@@ -49,12 +49,13 @@ fi
 file=`sed "s/[JNPES]-\([a-z_]*\)/\1/g" <<< $3`
 file="?-R*S*-*-$file.tex"
 file=`cd problems;ls $file`
-if [[ !($file =~ [a-z_]+) ]]; then
+# this form of regex is needed because [a-z] set depends on locale
+if [[ ! ( $file =~ ^[JNPES]-R[0123456789]+S[0123456789]-[0123456789]+-[abcdefghijklmnopqrstuvwxyz_]+\.tex$ ) ]]; then
```

Comment:
OK, what if we `export LC_COLLATE=C` did at the beginning of the script? All regexs would be simpler and I hope none of the git operations would be affected.